src/masternode/meta.h

```diff
@@ -96,6 +101,24 @@ class CMasternodeMetaInfo
     int64_t GetLastOutboundAttempt() const { return lastOutboundAttempt; }
     void SetLastOutboundSuccess(int64_t t) { lastOutboundSuccess = t; outboundAttemptCount = 0; }
     int64_t GetLastOutboundSuccess() const { return lastOutboundSuccess; }
+    bool SetPlatformBan(bool is_banned, int height)
+    {
+        LOCK(cs);
+        if (height <= m_platform_ban_updated) {
+            return false;
+        }
+        if (height == m_platform_ban_updated && !is_banned) {
```

Comment:
This condition is unreachable now because we'll bail out in `if (height <= m_platform_ban_updated) {` above (it was changed from `if (height < m_platform_ban_height) {` in https://github.com/dashpay/dash/pull/6613/commits/c739f3a6e001ce1a8928d81fd8bf0772d45e5d3e#diff-4b932ee0759e8f52676ccdd2f94fb3a39ee0652695edfa3822ecb3dcb7ef9f30R107)

Reply:
This is still confusing to me tbh... So this condition is basically "evonode CAN be banned at the height it was unbanned but it CAN NOT be unbanned at the height it was banned", right? We need a test for this edge case I guess.